{{cookiecutter.project_shortname}}/docker-services.yml

```diff
@@ -56,6 +55,14 @@ services:
   es:
     image: elasticsearch:5.6
```

Comment:
After the ES6 support, we can update this one to:

``docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2``

See https://www.elastic.co/guide/en/elasticsearch/reference/6.2/docker.html#_image_types

Reply:
waiting for ES6 to be released